oauth/localhost-key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDHDT/NcLhkTMn6
+M7gIcZEaQXDNZvYnUfS9UkvtdMpBDz2pbbYbQ1xB6RWJXi/0356SW3VMiF+unWeG
+iQd+QG3Mgu/0a9+s+AzFP3iyxp++1+5vxR7sKuy9at0jvvnhcUG09JXuVt9xv8ur
+HBQVkDDS+7WhPG2Ku6z6MRlPtqVsHgOgoui6vzCYfhkR2Ao3UwWDQEi51xZzE0Sr
+ZdedxRFscCJSpuT/+5lR+hrKMCUdXOh83qPD2aeC6BNBRIEoYJokAnMF0BQrd/jU
+S7Ez7l//zwfkKKLamoUhFbI627Phv/gE1+bmo92gp23FCHyDlFCSRV3R4iyLLLq8
+0YGzfy6hAgMBAAECggEAaTs46sxXvzbVZL/o4BXiB0Zncz+mjL0ILPqQB3ZB4slC
+6QqGRr2CzZmC+D9iDQwXkfpl9dL2+qJec8KHwpgfNApY/xf0hUYRwkm+7oyRMVyI
+wnAs2+AJjk4roNU615N/kEHp7bnhoQori0dp2siC91d4+0euBbUYSqCgaAMASzpY
+M0lAmeS36NigQ4sNwCHVKrJMxmFsYDq8wIuThSWD9oLYCQI2GfRP60hCIMuC/T+w
+1gBfalraZWWMAL4T7v8hVOgmg5gwpkwR21y8LOF3f31OQeduADw1dl8WKjsNXooL
+2YRXfRlmp7r8GAbS9yl0op17qUMTRswNFKnUfbyM3QKBgQDgiy95zzrSswIAeFsj
+fmsXPw+v/8k9l2IqtxbS+lk8DEMEihOMcsXr8DmPWfbz9PcX/eV0jvFpQhv8VUqM
+llZxmRvRsVEnjwnK7UnslMhPBErR7hUUH+9gehpn3SCwkNjMn5qUaeC81UExImQA
+mmBRLqDdMrWIwY60dkuvfzsvBwKBgQDi79iOrQ6exft4jocnneco+zbq/R0KUGPI
++I4HhaThtRjjnrGpT16kSgpGSUAHPEiKE1bODaI6isY99kxGxTiysr7DtGW2O7Sh
+ZMR1isfISMpAS+/IGS1doWYkuacvMZu7+515BEf5KPWHf6ZUzzp0+WQCyO4zTSK4
+k69JNEAjFwKBgHrlEtXrSrmZCcZyGQejID+33wEn0N+p5CuWy6IrLP3FhESOwcZD
+5vmMICTKS3QYJcj2yEf0GELQ0PpZQBUUDznbX7N0BgPNsMFk/+nBQKSmp8CgOt8p
+310BV0RzbXnVvSK5U+RY0ozndufKd8swsvibuaV3JI41ycIRYLkrvJuvAoGBAOJ1
+g75E9bz9Roe0eyzOhtm/brsqoGslc54eYuUguTu/Ezo5p4KUQJZjiENBX5KY1q5q
+xAnVUxlmWJOGj/FjZ8PzWShEgedLR2JzAxVkyx6TfLiSzpH4hevUXvCxDyOqNoVt
+fmRAIG6EMBfqCxrAvM3z/JhG2GT7SXW1corziDk7AoGAJ6j3xwAdDpQE57CXq2gr
+x4HJIKa6GSZ2o2ygC6k46o3QG5C4ITKPjsUOKI0Wjyfu5YGYFK/K6RYmLX5Ei8xf
+01EuEkW+9jhZLESs/pMsPiKL3nILAxY2LJQbgZWhnCuCb8q6V+NRRjOJd/finRxR
+Jhtn3ZU3bWYlsArREwcfV7Y=
+-----END PRIVATE KEY-----
```

Comment:
I think Cert and Key shouldn't be part of the repo and instead be generated on the user's system.